after/ftplugin/tex_matchup.vim

```diff
@@ -97,7 +97,11 @@ function! s:get_match_words()
   endif
 
   " dollar sign math
-  let l:match_words .= ',\$:\$\g{syn;!texMathZoneX}'
+  if exists('b:vimtex')
```

Comment:
note: unlike https://github.com/andymass/vim-matchup/commit/ee1d3e033bc22c2079e950e39d59631573922f79, I'm using `b:vimtex` variable to check whether vimtex is used.

Reply:
Yes, that looks like a good choice to me.

Reply:
`if s:has_plugin('vimtex') && exists('*execute') && execute(':scriptnames') =~# 'vimtex.*ftdetect.*tex\.vim'` seems much more complicated than necessary!